mario/core/AttrData.py

```diff
@@ -315,6 +315,62 @@ def to_iot(
             ),
         )
 
+    def to_chenery_moses(
+        self, inplace=True, scenarios=None,
+    ):
+
+        """The function will transform an Isard SUT table to a Chenery-Moses SUT table
+
+
+        .. note::
+
+            Calling this function will modify all scenarios in the database
+
+        Parameters
+        ----------
+
+        inplace : boolean
+            if True, implements the changes on the Database else returns
+            a new object without changing the original Database object
+
+        scenarios : str, list
+            if None, will implement the changes on all the scenarios, else will
+            implement the changes on the given scenarios
+        
+        Returns
+        -------
+        None :
+            if inplace True
+
+        mario.Database :
+            if inplace False
+        """
+        if not inplace:
+            new = self.copy()
+            new.to_chenery_moses(inplace=True)
```

Comment:
you need to pass the scenarios also here, to make sure that with inplace=False, the same would happen

Reply:
Thanks, you are right I added scenarios=None in line 350